src/black/__init__.py

```diff
@@ -90,10 +90,10 @@
 from blib2to3.pgen2 import token
 from blib2to3.pytree import Leaf, Node
 
-if TYPE_CHECKING:
-    from concurrent.futures import Executor
-
 COMPILED = Path(__file__).suffix in (".pyd", ".so")
+DEFAULT_WORKERS: Final = os.cpu_count()
```

Comment:
Hm this does a syscall at startup, we may want to put it into a cached-forever function instead.

Reply:
Though then again

```
In [16]: %timeit os.cpu_count()
1.05 µs ± 10.1 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```